publish/docker-setup.sh

```diff
@@ -4,36 +4,32 @@
 
 set -eou pipefail
 
-apt-get update
-apt-get install \
+apt update
+apt install \
     apt-transport-https \
     curl \
     git \
     gnupg \
     wget \
+    ca-certificates \
+    unzip \
+    zip \
     -y --no-install-recommends
 
-# setup install for adoptopenjdk
-# https://adoptopenjdk.net/installation.html#linux-pkg-deb
-wget -qO - https://adoptopenjdk.jfrog.io/adoptopenjdk/api/gpg/key/public | apt-key add -
-echo "deb https://adoptopenjdk.jfrog.io/adoptopenjdk/deb $(
```

Comment:
This listing is abandoned since 2021 and has no recent releases, replacement steps below are from current instructions